go.mod

```diff
@@ -1,38 +1,82 @@
 module github.com/metallb/metallb-operator
 
-go 1.16
+go 1.17
```

Comment:
just checking, did you run a go mod tidy after this?

Reply:
yes, i did. should we commit this so that go 1.16 checks might completely go away ?